### DISCUSSION TITLE
Ant BMS

### DISCUSSION THREAD

**Discussion by Edser81** [Louisvdw/dbus-serialbattery#272] [Q&A] [answered]
So i was able to connect ant BMS but 2 problems. I have 2 BMS/2battery banks 200A BMSes when i connected the first one it worked i was all excited i could see highest cell volt lowest cell volt AH and battery % plus whats going in and out of battery. As soon as i connected the second battery that one was identified as a serialBattery (Sinowealth) and got lots of low voltage high temp notifications. I then disconnected it and tried again same thing tried checking wires and all same. But second problem is my first battery BMS i installed gives me a Low Cell Voltage warning showing 0.000v i get this message every 1-2mins. I have checked all setting even tried downgrading from 2.92 to 2.91 same results. also Tried version using the new v.14 but same. I also cant seem to get it to work without having the LCD connected that's weird so i have both LCD and USB cable TTL connected and seems to id one Battery correctly but not the other. been at it all day lol. Thanks in advanced.

**Comment by Louisvdw** [comment]
You will need to look at the log files for each of your ANT BMS devices. This should tell us why the second one fails. 
https://github.com/Louisvdw/dbus-serialbattery/wiki/Troubleshoot

There is a newer protocol for the ANT BMS that is not yet supported #262 but I would expect that you have to of the same models?

**Comment by Edser81** [comment]
ok so being a newbie at this on the troubleshoot page it says 2 important log files to look at /data/log/serial-starter/current But how do i get there? Do i remove Micro card and loog at them on comp or use putty? also my BMS is 3 years old atleast maybe 4. Just never tried this since didnt even know this was possible. today i woke up t find the first ant BMS switched to (Sinowealth) reset everything again and connected it again its ok except that low voltage  and after 10-15 mins of leaving it plugged it changes from ant to sinowealth again. Yes i have 2 of the same models but one i have changed the name and serial on the BMS to be able to connect it on Bluetooth had a problem 3 years ago when i got them they were with same Serial and couldnt get both online. So changing this showed both. Thanks for the reply looking into how to do the protocol.

**Comment by Edser81** [comment]
Update i was able to get to log files and doing the tail -f /data/log/dbus-serialbattery.ttyUSB0/current
it keeps only detecting my ANT BMs as sinowealth. any way to dissable and use only ant?

**Comment by Edser81** [comment]
![Screenshot 2022-10-23 095814](https://user-images.githubusercontent.com/115951864/197405208-bcc8905b-17ff-4d5f-a6c1-a2a88054c87b.jpg)
 this is what i see when it connects to ant
![Screenshot 2022-10-23 113204](https://user-images.githubusercontent.com/115951864/197409570-bf4f1fa0-b4ec-4b9d-b9e8-991469e6917c.jpg)
![Screenshot 2022-10-23 113340](https://user-images.githubusercontent.com/115951864/197409571-2d9b522e-e731-4ac9-be5a-2bc60a5bd979.jpg)
Thats where i am at i created a second ANT1.PY with version 2.1 to display second one. seems to work but only to monitor both since i keep getting back and forth disconnect form one and low cell voltage warning even when just one is on any ideas how to get rid of low cell voltage notifications i have no low cells thanks.

**Comment by Louisvdw** [comment, marked as answer]
It looks like your one ANT on USB0 connects and then get many incorrect replies. This normally happens when there is a loose connection or interference in the cable. Make sure your connection is secure. You can also try to use shielded wires. (a little foil over your wire can also help but this has a short/fire risk)

When the ANT disconnects or fails to connect it will try the other drivers as well. It seems that the Sinowealth is very similar to the connection test of the ANT which is why it is then sometimes picked up. We have made some changes to disable some drivers in future, but in the currect release you need to comment out the Sinowealth with a # in the dbus-serialbattery.py file. There are some others commented out already so just follow that example. Then reboot the GX for it to apply.
